METRICS.MD

```diff
@@ -5,21 +5,21 @@
 * `active_record.users.find`
```

Comment:
There is a bunch of outdated information in here, and I think when the README links to the production metrician documentation, this file is probably redundant and can be deleted entirely. Thoughts?